feature.js

```diff
@@ -0,0 +1,108 @@
+  role: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Magnam fuga rem sequi dolore',
+  img: 'img/CZ-binance.jpg',
+};
+
+const lastSpk = {
+  name: 'Brian Armstrong',
+  position: 'CEO of Coinbase.',
+  role: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Magnam fuga rem sequi dolore',
+  img: 'img/brian-armstrong-coinbase.jpg',
+};
+
+const spkArr = [firstSpk, secondSpk, thirdSpk, fourthSpk, fifthSpk, lastSpk];
+
+window.addEventListener('load', () => {
+  const fspCont = document.querySelector('#fsp');
+
+  fspCont.innerHTML += `<h3 class="feat">Featured speakers</h3>
+    <div class="sub-cont">
+       <div class="hold">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[0].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[0].name}</h3>
+                  <i class="position">${spkArr[0].position}</i>
+                  <p class="role">${spkArr[0].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[1].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[1].name}</h3>
+                  <i class="position">${spkArr[1].position}</i>
+                  <p class="role">${spkArr[1].role}</p>
+               </div>
+            </div>
+       </div>
+       <span id="break"></span>
+       <div class="hold activ">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[2].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[2].name}</h3>
+                  <i class="position">${spkArr[2].position}</i>
+                  <p class="role">${spkArr[2].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[3].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[3].name}</h3>
+                  <i class="position">${spkArr[3].position}</i>
+                  <p class="role">${spkArr[3].role}</p>
+               </div>
+            </div>
+       </div>
+
+       <div class="hold activ">
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[4].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[4].name}</h3>
+                  <i class="position">${spkArr[4].position}</i>
+                  <p class="role">${spkArr[4].role}</p>
+               </div>
+            </div>
+            <div class="speaker">
+               <img class="spk-img" src="${spkArr[5].img}">
+               <div class="spk-text">
+                  <h3 class="name">${spkArr[5].name}</h3>
+                  <i class="position">${spkArr[5].position}</i>
+                  <p class="role">${spkArr[5].role}</p>
+               </div>
+            </div>
+       </div>
+   </div>
+    `;
+});
```

Comment:
You should have a function that iterates over your `spkArr` so that it dynamically renders the speaker's information.

<img width="862" alt="Screen Shot 2022-07-27 at 9 33 47" src="https://user-images.githubusercontent.com/82426682/181274556-bfc6eff6-0312-44fd-a436-a45866da791a.png">

Please refactor your code so it iterates over the array mentioned above.